R/geom-ribbon.r

```diff
@@ -22,8 +22,8 @@
 #' @inheritParams layer
 #' @inheritParams geom_bar
 #' @param outline.type Type of the outline of the area; `"both"` draws both the
-#'   upper and lower lines, `"upper"` draws the upper lines only. `"legacy"`
-#'   draws a closed polygon around the area.
+#'   upper and lower lines, `"upper"`/`"lower"` draws the either lines only.
```

Comment:
Something seems to have gone wrong here, "draws the either lines only" doesn't make sense. Maybe "draws the respective lines only"?

Reply:
```suggestion
#'   upper and lower lines, `"upper"`/`"lower"` draws the respective lines only.
```